README.md

```diff
@@ -217,6 +217,18 @@ you immediately prior to rebooting a node:
 We recommend setting `--slack-username` to be the name of the
 environment, e.g. `dev` or `prod`.
 
+### Generinc Notifications 
```

Comment:
I would just call it "Notifications".

When adding the `--slack*` deprecation, we can then remove the whole `Slack Notifications` section, so that people are not enclined to use it anymore.

Reply:
Done